recipes/eigen/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "3.2.8" %}
+{% set filename = "eigen-%s.tar.gz" % version %}
+
+package:
+  name: eigen
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}
```

Comment:
Going to revert back to the old way of setting the filename. Just not thinking we are getting much from this.